### PR TITLE
Add core weekly 5 2021

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

File: news/_posts/core-weekly/2021-02-08-coreweekly-05-2021.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+* [#25](https://github.com/PrestaShop/ps_dataprivacy/pull/25): Release 2.0.1, by [@Progi1984](https://github.com/Progi1984)
+
+
+### Theme customization module
+* [#37](https://github.com/PrestaShop/ps_themecusto/pull/37): Release 1.2.1, by [@Progi1984](https://github.com/Progi1984)
+
+
+## Where to start contributing?
+
+What about [sending test emails in module ps_reminder](https://github.com/PrestaShop/PrestaShop/issues/18569)? This is a feature request submitted last year by long-time contributor [@PrestaShark](https://github.com/PrestaShark), and it is one of our [good first issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
```
Comment (Progi1984) [inline news/_posts/core-weekly/2021-02-08-coreweekly-05-2021.md]: :warning: Warning, the ps_reminder module is going to be archived.

**Review by Progi1984 — COMMENTED** [review]
A small comment

File: news/_posts/core-weekly/2021-02-08-coreweekly-05-2021.md (hunk 1 above)
Comment (matks) [reply news/_posts/core-weekly/2021-02-08-coreweekly-05-2021.md]: Ah, thanks for the tip. I'm going to update the issue and change the Core Weekly.

Comment (matks) [reply news/_posts/core-weekly/2021-02-08-coreweekly-05-2021.md]: Fixed in https://github.com/PrestaShop/prestashop.github.io/pull/841/commits/460bda91878bce2eb3b4b56dbc2fa9b3e185cea6

**Review by Progi1984 — APPROVED** [review]

**Comment by matks** [comment]
Thank you for review !

**Pull request merged**